gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/GradebookPage.java

```diff
@@ -648,4 +646,46 @@ private Label constructTableLabel(final String componentId, final DataTable tabl
 
 		return label;
 	}
-}
\ No newline at end of file
+
+
+	/**
+	 * Comparator class for sorting Assignments in their categorised ordering
+	 */
+	class CategorizedAssignmentComparator implements Comparator<Assignment> {
```

Comment:
We have a bunch of comparators in the service, and now one here, maybe these should be split out into the business.util package?